Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -128,274 +244,186 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var sendsNotifications: Bool = true
     
-    /**
-     Shows a button that allows drivers to report feedback such as accidents, closed roads,  poor instructions, etc. Defaults to `true`.
-     */
-    public var showsReportFeedback: Bool = true {
-        didSet {
-            ornamentsController?.reportButton.isHidden = !showsReportFeedback
-            showsEndOfRouteFeedback = showsReportFeedback
-        }
-    }
+    private var traversingTunnel = false
+    private var approachingDestinationThreshold: CLLocationDistance = 250.0
+    private var passedApproachingDestinationThreshold: Bool = false
+    private var currentLeg: RouteLeg?
+    private var foundAllBuildings = false
     
-    /**
-     Shows End of route Feedback UI when the route controller arrives at the final destination.
-     
-     Defaults to `true.`
-     */
-    public var showsEndOfRouteFeedback: Bool {
-        get {
-            arrivalController?.showsEndOfRoute ?? false
-        }
-        set {
-            arrivalController?.showsEndOfRoute = newValue
-        }
-    }
+    // MARK: Additional Functionality
```

Comment:
This is pretty vague, sort of like a junk drawer for the class. We could replace this with a bunch of more specific marks, one for each member.